package.json

```diff
@@ -26,8 +26,8 @@
   "license": "MIT",
   "dependencies": {
     "bindings": "^1.2.1",
-    "libxmljs-mt": "0.16.1",
-    "nan": "^2.1.0"
+    "libxmljs-mt": "0.18.0",
```

Comment:
don't merge yet, travis throws this error:
../../nan/nan.h:43:3: error: #error This version of node/NAN/v8 requires a C++11 compiler

maybe it was wrong to also try to update libxmljs-mt in this pr